backend_addon/cookiecutter.json

```
@@ -46,5 +56,8 @@
     "cookieplone.filters.package_path"
   ],
   "__cookieplone_repository_path": "",
-  "__cookieplone_template": ""
+  "__cookieplone_template": "",
```

```suggestion
  "__cookieplone_template": "backend_addon",
```

In my test without this change it generated the following:
```
This was generated by [the cookieplone-templates  template](https://github.com/plone/cookieplone-templates/tree/main/) on 2025-03-25 02:52:19. A special thanks to all contributors and supporters!
```

---

I added this, but also refactored the tests to make sure variables from extra-context are always added

---

@ericof Ah, so that's where it was supposed to come from. Makes sense.